src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAGIC_NUMBER = 100;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        int numberRandom = random.nextInt(MAGIC_NUMBER);
+        String colorRandom = colorSupplier.getRandomColor();
```

Comment:
actually these variables are redundant as they are not used anywhere else. You can directly pass the result of random and colorSupplier methods calls to the ball's setters